ipn/ipnlocal/local.go

```diff
@@ -156,11 +156,13 @@ func RegisterNewSSHServer(fn newSSHServerFunc) {
 	newSSHServer = fn
 }
 
-// watchSession represents a WatchNotifications channel
+// watchSession represents a WatchNotifications channel,
+// an [ipnauth.Actor] that opened the session
 // and sessionID as required to close targeted buses.
 type watchSession struct {
 	ch        chan *ipn.Notify
-	sessionID string
+	actor     ipnauth.Actor
+	sessionID string // ID of the watchSession; not to be confused with an [ipnauth.SessionID]
```

Comment:
This is the confusion I'm worried about. I'm very skeptical we need both of these session types.

Why can't the LocalBackend keep track of which Watch sessionID is which actor and send to all watches that match?

Reply:
Ideally, I think we should remove the `watchSession`-specific `sessionID` and always use the actor and its session.
I started with that initially, but it required too many changes specific to `tailscale serve`, which I think should be worked on and reviewed separately.

Reply:
I renamed the new `SessionID` type to `ClientID` and removed the comment.